index.html

```diff
@@ -383,7 +383,7 @@ <h3>Ed25519VerificationKey2020</h3>
               "id": "https://example.com/issuer/123#key-0",
               "type": "Ed25519VerificationKey2020",
               "controller": "https://example.com/issuer/123",
-              "publicKeyMultibase": "zgo4sNiXwJTbeJDWZLXVn9uTnRwgFHFxcgDePvEC9TiTYgRpG7q1p5s7yRAic"
+              "publicKeyMultibase": "zdbDmZLTWuEYYZNHFLKLoRkEX4sZykkSLNQLXvMUyMB1"
```

Comment:
```suggestion
              "publicKeyMultibase": "z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"
```

https://github.com/w3c-ccg/did-method-key/blob/master/test-vectors/ed25519-x25519.json#L2

Reply:
I got `z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP` for this keypair (`#key-0`) in multibase/multicodec format; in #11: https://github.com/w3c-ccg/lds-ed25519-2020/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R520

Reply:
yeah, the examples should probably be updated consistently, I am just pointing out the obvious issue with it needing to look like: `z6Mk`